### PR TITLE
Prototype RedHat.Container package.

### PR DESCRIPTION
**Pull request by tmds**
This package facilitates using Red Hat base images and interacting with the OpenShift internal image registry from the .NET SDK.

### CODE REVIEW

**Comment by tmds** [comment]
@baronfel if you are interested, can you take a look at this PR?

Besides taking a look at the implementation, may be you also have some feedback on how I'm (ab)using the csproj file to create a NuGet package that actually doesn't have C# related build output.

File: src/RedHat.Container/RedHat.Container.csproj [hunk 1]
```diff
@@ -0,0 +1,28 @@
+<Project Sdk="Microsoft.NET.Sdk">
```
Comment (baronfel) [inline src/RedHat.Container/RedHat.Container.csproj:1]: I think you could use the NoTargets SDK here and skip a lot of the workarounds you had to do.

MSBuildSdks/src/NoTargets/README.md at main · microsoft ... - GitHub https://github.com/microsoft/MSBuildSdks/blob/main/src/NoTargets/README.md

**Review by baronfel — APPROVED** [review]
This looks great! It's basically exactly what I was hoping for in terms of extensibility for anyone wanting to override the SDK defaults.

**Comment by tmds** [comment]
@baronfel thanks for taking a look, and the suggestion on using the NoTargets SDK!

**Pull request merged**